migtests/lib/yb.py

```diff
@@ -149,3 +149,14 @@ def invalid_index_present(self, table_name, schema_name):
 				return True
 
 		return False
+
+
+	def fetch_all_triggers(self, schema_name="public") -> List[str]:
+		cur = self.conn.cursor()
+		cur.execute(f"SELECT trigger_name FROM information_schema.triggers WHERE trigger_schema = '{schema_name}'")
+		return [trigger[0] for trigger in cur.fetchall()]
+
+	def fetch_all_procedures(self, schema_name="public") -> List[str]:
```

Comment:
Rename to `list_procedure_names(...)` and move it after the `list_trigger_names(...)`.